src/util/BrowserFeatures.js

```diff
@@ -58,6 +58,14 @@ define(['underscore'], function (_) {
     return /(msie|trident)/i.test(navigator.userAgent);
   };
 
+  fn.isFirefox = function () {
+    return navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
+  };
+
+  fn.isChrome = function () {
+    return /Chrome/.test(navigator.userAgent) && /Google Inc/.test(navigator.vendor);
+  };
```

Comment:
@anishashetty-okta or @rchild-okta can you just double check that this is the best way to identify Chrome and Firefox?